include/PR/asm.h

```diff
@@ -1,72 +1,124 @@
-#ifndef PR_ASM_H
-#define PR_ASM_H
+#ifndef ASM_H
```

Comment:
Not a big deal but could you add the `PR_` prefix back. Its what we have been doing for all header files in PR.